python/pyiceberg/io/pyarrow.py

```diff
@@ -278,6 +279,15 @@ def _get_fs(self, scheme: str) -> FileSystem:
                 "region": self.properties.get(S3_REGION),
             }
             return S3FileSystem(**client_kwargs)
+        elif scheme in {"gs", "gcs"}:
+            if access_token := self.properties.get("gs.access-token"):
```

Comment:
Can we add `gs.credential-token-expiration` to the docs?

Reply:
I missed it. I've added it to the docs now.